src/server/views/dashboard/list-item-confirm/layout.njk

```diff
@@ -1,54 +1,55 @@
 {% from "button/macro.njk" import govukButton %}
-{% extends "./dashboard.njk" %}
+{% extends "./../dashboard.njk" %}
+
+{% block pageTitle %}
+  List management - {{ _.startCase(action) }} - {{ title + " - " if title }} {{ SERVICE_NAME }}
+{% endblock %}
 
 {% block aside %}
+  <div class="back-links__item">
+    <a class="govuk-link govuk-back-link" href="{{ listItemUrl }}">Back</a>
+  </div>
 {% endblock %}
 
 {% block dashboard %}
 
-  <form class="form" action="{{ listItemUrl }}/delete" method="post" required>
+  <form class="form" method="post">
     <input type="hidden" name="_csrf" id="_csrf" value="{{ csrfToken }}" />
-    <input type="hidden" name="action" value="remove"/>
 
     <h1 class="govuk-heading-l">
-      Remove this service provider
+      {% block heading %}
+        {{ _.startCase(action)}} this service provider
+      {% endblock %}
     </h1>
 
     <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
 
-    <h2 class="govuk-heading-m">You are about to remove the following provider from the live list:</h2>
+    {% block confirmChangesContent %}
+    {% endblock %}
 
+    <p class="govuk-body">An email with this request will be sent to the following service provider:</p>
```

Comment:
most of this content was the same across all of the individual template. 

for the templates that extend this layout.njk, you can do
  
```
{% block confirmChangesContent %}
  the bespoke stuff I want to insert
{% endblock %}
```

Reply:
correct me if I'm wrong but I think this will lead to a change in content layout so the content below the provider details on lines 31-36 (previously 21-26) will appear above it.  In the diff I'm referring to lines 28-32 that have now been removed.  No issues but have you checked this with Neil from a content perspective?

Reply:
sorry yep, should go below

Reply:
the diff got WEIRD.. it is showing this was previously the list-item-confirm-remove template. 

I'll check the order. 

It looks like the content should be in this order:
- h1 $action this provider
- confirmChangesBlock
  - h2 you are about to do something
  - An email will be sent to service provider -> looks like this is only applicable in some actions
  - service provider details  -> looks like this is only applicable in some actions
- record your decision